docs/3-setting_up_the_configuration.md

```diff
@@ -0,0 +1,80 @@
+Step 3 - Setting up the configuration
+=====================================
+
+### A - Default configuration
+
+```yaml
+# app/config/packages/ticketing.yaml
+ticketing:
+
+    # The name of the default status
+    default_status_name:  open
+
+    # Enable the seen/unseen display for tickets
+    enable_history:       true
+
+    # Set a role that will have access to the "staff" tickets
+    restricted_tickets_role: ROLE_USER
+
+    # if true, all new tickets will only be accessible to the owner and the restricted_ticket_role. If false, all new tickets will be public
+    enable_ticket_restriction: true
+    entities:
+        ticket:               App\Entity\Ticket
+        ticket_category:      App\Entity\TicketCategory
+        ticket_comment:       App\Entity\TicketComment
+        ticket_history:       App\Entity\TicketHistory
+        ticket_keyword:       App\Entity\TicketKeyword
+        ticket_status:        App\Entity\TicketStatus
+    templates:
+        layout:               '@Ticketing/base.html.twig'
+        index:                '@Ticketing/ticketing/index.html.twig'
+        new:                  '@Ticketing/ticketing/new.html.twig'
+    assets:
+        stylesheets:
+            - bundles/ticketing/vendor/css/bootstrap.css
+            - bundles/ticketing/vendor/css/bootstrap-datepicker3.css
+            - bundles/ticketing/vendor/css/font-awesome.css
+            - bundles/ticketing/vendor/css/ionicons.css
+            - bundles/ticketing/vendor/css/select2.css
+            - bundles/ticketing/css/AdminLTE.css
+            - bundles/ticketing/css/helper.css
+            - bundles/ticketing/css/skin-ticketing.css
+        javascripts:
+            - bundles/ticketing/vendor/js/jquery.js
+            - bundles/ticketing/vendor/js/bootstrap.js
+            - bundles/ticketing/vendor/js/adminlte.js
+            - bundles/ticketing/vendor/js/bootstrap-datepicker.js
+            - bundles/ticketing/vendor/js/bootstrap-datepicker.fr.js
+            - bundles/ticketing/vendor/js/select2.js
+            - bundles/ticketing/vendor/js/select2-fr.js
+            - bundles/ticketing/js/script.js
+```
+
+### B - Parameters explaination
+
+All parameters are optional if you use the bundle the basic way.
+
+But if needs be you can override the default configuration using the the full configuration above.
```

Comment:
I have difficulties to understand this. I think a break with "," should be help.